pgxn/neon/neon.c

```diff
@@ -366,23 +366,38 @@ RestoreRunningXactsFromClog(CheckPoint *checkpoint, TransactionId **xids, int *n
 	if (!TransactionIdIsValid(checkpoint->oldestActiveXid))
 	{
 		elog(LOG, "cannot restore running-xacts from CLOG because oldestActiveXid is not set");
```

Comment:
Shouldn't this message be under `if (running_xacts_overflow_policy == OP_WAIT)` now? Otherwise, now we have a path to write this message, but then still try to restore running xacts (`if (TransactionIdIsValid(checkpoint->oldestXid))` path)

Reply:
I specially move the message before check to register I the go the fact that subxids overflow took place.